### PR TITLE
Fix javadoc for AbstractApplicationContext.registerBeanPostProcessors()

### PR DESCRIPTION
**Pull request by rupertw**
Fix explanation in javadoc of `AbstractApplicationContext#registerBeanPostProcessors`.

### CODE REVIEW

File: spring-context/src/main/java/org/springframework/context/support/AbstractApplicationContext.java [hunk 1]
```diff
@@ -713,9 +713,8 @@ protected void invokeBeanFactoryPostProcessors(ConfigurableListableBeanFactory b
 	}
 
 	/**
-	 * Instantiate and invoke all registered BeanPostProcessor beans,
+	 * Instantiate and register all BeanPostProcessor beans,
 	 * respecting explicit order if given.
-	 * <p>Must be called before any instantiation of application beans.
```
Comment (sbrannen) [inline spring-context/src/main/java/org/springframework/context/support/AbstractApplicationContext.java]: Please reinstate this line.

**Review by sbrannen — CHANGES_REQUESTED** [review]
Please make the requested change.

File: spring-context/src/main/java/org/springframework/context/support/AbstractApplicationContext.java (hunk 1 above)
Comment (rupertw) [reply spring-context/src/main/java/org/springframework/context/support/AbstractApplicationContext.java]: Do you mean to make a new commit to reinstate the line?

Comment (rupertw) [reply spring-context/src/main/java/org/springframework/context/support/AbstractApplicationContext.java]: The line is already reinstated.

**Review by sbrannen — APPROVED** [review]

**Comment by sbrannen** [comment]
This has been merged into `master`.

Thanks

**Pull request merged**